future-proofing.yaml

```diff
@@ -23,31 +23,50 @@ future-proofing:
         - composer require rector/rector --dev
         - vendor/bin/rector init
 
-        # section that registers the rectors
-        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-#        - bin/future add-rule InlineConstructorDefaultToPropertyRector # just an example
-        # /section that registers the rectors
+        # section that registers rector sets
+        - bin/future add-set \Rector\Set\ValueObject\SetList::CODE_QUALITY
```

Comment:
@calinpristavu is it possible to make this command generate something like this? (on subsequent calls)

![image](https://user-images.githubusercontent.com/20539991/203614843-52de2182-316e-4cd0-b2cd-d45585d1f51e.png)

Reply:
Already done, with the mention that each ruleset needs to have a FQCN

Reply:
It does